maxmindupdater/__main__.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
+from argparse import ArgumentParser
+from maxmindupdater import update_db
+
+
+def main():
+    parser = ArgumentParser()
+    parser.add_argument('db_path')
+    parser.add_argument('license_key')
+    parser.add_argument('edition_id')
```

Comment:
Just did a test and metavars simply replace the names of positional arguments. So if I don't use a different name for the metavar they serve no purpose.